src/components/pages/RoleSignup/Applications/Mentor.js

```diff
@@ -474,14 +474,13 @@ const Mentor = ({ dispatch, error, successPage }) => {
                     icon: <InfoCircleOutlined />,
                   }}
                 ></Form.Item>
-                <Form.Item
+                <Input.TextArea
```

Comment:
good catch. This SHOULD mirror the mentee issue so thanks for putting them in alignment